Gradle build fails due to missing dependency

The build fails due to a missing dependency for `org.eclipse.gemini:org.eclipse.gemini.blueprint.extender:1.0.2.RELEASE`

Maven central currently contains only the 2.0.0.RELEASE:
http://repo1.maven.org/maven2/org/eclipse/gemini/blueprint/gemini-blueprint-extender/

```bash
 $ ./gradlew

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred evaluating project ':blueprint-override'.
> Could not resolve all dependencies for configuration ':blueprint-override:extender'.
   > Could not find org.eclipse.gemini:org.eclipse.gemini.blueprint.extender:1.0.2.RELEASE.
     Searched in the following locations:
         https://repo1.maven.org/maven2/org/eclipse/gemini/org.eclipse.gemini.blueprint.extender/1.0.2.RELEASE/org.eclipse.gemini.blueprint.extender-1.0.2.RELEASE.pom
         https://repo1.maven.org/maven2/org/eclipse/gemini/org.eclipse.gemini.blueprint.extender/1.0.2.RELEASE/org.eclipse.gemini.blueprint.extender-1.0.2.RELEASE.jar
         https://artifacts.alfresco.com/nexus/content/groups/public/org/eclipse/gemini/org.eclipse.gemini.blueprint.extender/1.0.2.RELEASE/org.eclipse.gemini.blueprint.extender-1.0.2.RELEASE.pom
         https://artifacts.alfresco.com/nexus/content/groups/public/org/eclipse/gemini/org.eclipse.gemini.blueprint.extender/1.0.2.RELEASE/org.eclipse.gemini.blueprint.extender-1.0.2.RELEASE.jar
         https://jcenter.bintray.com/org/eclipse/gemini/org.eclipse.gemini.blueprint.extender/1.0.2.RELEASE/org.eclipse.gemini.blueprint.extender-1.0.2.RELEASE.pom
         https://jcenter.bintray.com/org/eclipse/gemini/org.eclipse.gemini.blueprint.extender/1.0.2.RELEASE/org.eclipse.gemini.blueprint.extender-1.0.2.RELEASE.jar
     Required by:
         com.github.dynamicextensionsalfresco:blueprint-override:1.4.2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

--- a/build.gradle
+++ b/build.gradle
@@ -84,6 +84,12 @@ subprojects {
             url "https://artifacts.alfresco.com/nexus/content/groups/public/"
         }
         jcenter()
+        maven {
+            url "http://repository.springsource.com/maven/bundles/release"
+        }
+        maven {
+            url "http://repository.springsource.com/maven/bundles/external"
+        }
     }
 
     group = 'com.github.dynamicextensionsalfresco'

